### PR TITLE
Blue Sec Update Two: Seclectric Bluegaloo

### PR DESCRIPTION
**Pull request by BoskiYourk**
## About the PR

- Added a bluesuit outfit for the detective
- Added a new bluesuit jumpsuit for the warden
- Added new hats to match the officer blue suit
- Added everything to the loadout menus and uniform printer

## Why / Balance

HoS and officers get blue suits, so why not the warden and detective too?

### _Harmony Namescape
Resources/Prototypes/_Harmony/Entities/Clothing/Head/hats.yml -- Adds the beret and fedora entities
Resources/Prototypes/_Harmony/Entities/Clothing/Head/soft.yml -- Adds the blue cap entity
Resources/Prototypes/_Harmony/Entities/Clothing/OuterClothing/coats.yml -- Adds the blue coat entity
Resources/Prototypes/_Harmony/Entities/Clothing/Uniforms/jumpsuits.yml -- Adds the blue warden and det suit entities
Resources/Prototypes/_Harmony/Loadouts/Jobs/Security/detective.yml -- Detective loadout options
Resources/Prototypes/_Harmony/Loadouts/Jobs/Security/security_officer.yml -- Officer loadout options
Resources/Prototypes/_Harmony/Loadouts/Jobs/Security/warden.yml -- Warden loadout options
Resources/Prototypes/_Harmony/Recipes/Lathes/clothing.yml -- Lathe recipes 
Resources/Textures/_Harmony/Clothing/Head/Hats/beret_security_blue.rsi -- Beret sprite location
Resources/Textures/_Harmony/Clothing/Head/Hats/bluefedora.rsi -- Fedora sprite location
Resources/Textures/_Harmony/Clothing/Head/Soft/secbluesoft.rsi -- Cap sprite location
Resources/Textures/_Harmony/Clothing/OuterClothing/Coats/detective_blue.rsi -- Coat sprite location
Resources/Textures/_Harmony/Clothing/Uniforms/Jumpsuit/det_blue.rsi -- Detective suit sprite location
Resources/Textures/_Harmony/Clothing/Uniforms/Jumpsuit/warden_blue.rsi -- Warden suit sprite location

### Upstream Files
Resources/Prototypes/Catalog/VendingMachines/Inventories/detdrobe.yml -- Added the detective clothes to the detdrobe
Resources/Prototypes/Catalog/VendingMachines/Inventories/secdrobe.yml -- Added the beret and cap to the secdrobe
Resources/Prototypes/Entities/Structures/Machines/lathe.yml -- IDs for clothing printer
Resources/Prototypes/Loadouts/loadout_groups.yml -- Adds the new items to the loadout menu

## Media

Blue Det
Blue Ward 2
Blue Hat
Blue Beret

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
- add: Finished adding blue sec options for the rest of the security team.

### CODE REVIEW

**Comment by BoskiYourk** [comment]
Not sure what's up with the HoS hat showing up as having some bits removed, so hopefully that isn't too big of a deal...

**Review by DieselMohawk — CHANGES_REQUESTED** [review]
Fix the icon size for the blue beret. That's likely causing the test failure.

File: Resources/Textures/_Harmony/Clothing/Head/Hats/beret_security_blue.rsi/icon.png
Comment (DieselMohawk) [inline Resources/Textures/_Harmony/Clothing/Head/Hats/beret_security_blue.rsi/icon.png:1]: This is the offending file, please fix.

Change from 64x64 to 32x32

Comment (DieselMohawk) [reply Resources/Textures/_Harmony/Clothing/Head/Hats/beret_security_blue.rsi/icon.png:1]: On further inspection ALL of your icon files are in 64x64, please make them all 32x32.

File: Resources/Prototypes/_Harmony/Loadouts/Jobs/Security/warden.yml
Comment (DieselMohawk) [inline Resources/Prototypes/_Harmony/Loadouts/Jobs/Security/warden.yml:1]: "Jumpsuit" misspelled as "Jumsuit" may br causing errors

File: Resources/Prototypes/_Harmony/Entities/Clothing/Uniforms/jumpsuits.yml
Comment (DieselMohawk) [inline Resources/Prototypes/_Harmony/Entities/Clothing/Uniforms/jumpsuits.yml:1]: Please replace all instances of "jumsuit" with "jumpsuit"

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/secdrobe.yml
Comment (DieselMohawk) [inline Resources/Prototypes/Catalog/VendingMachines/Inventories/secdrobe.yml:1]: May be worthwhile to capitalize "Soft"

**Comment by DieselMohawk** [comment]
Damn, test failure, time to look at it again

**Review by FluffMe — CHANGES_REQUESTED** [review]
```
Resources/Textures/_Harmony/Clothing/OuterClothing/Coats/detective_blue.rsi: PNG not defined in metadata: Inhand-right.png
Resources/Textures/_Harmony/Clothing/OuterClothing/Coats/detective_blue.rsi: PNG not defined in metadata: Equipped-OUTERCLOTHING.png
Resources/Textures/_Harmony/Clothing/OuterClothing/Coats/detective_blue.rsi: equipped-OUTERCLOTHING: failed to open state equipped-OUTERCLOTHING.png
Resources/Textures/_Harmony/Clothing/OuterClothing/Coats/detective_blue.rsi: inhand-right: failed to open state inhand-right.png
```
While files for states are present in RSI, the states and file names are case sensitive.

Inhand-right.png and Equipped-OUTERCLOTHING.png should be renamed (uncapitalize first letter of the file name).

**Review by DieselMohawk — DISMISSED** [review]
Turns out I forgot to send my review, looks good now though

**Comment by DieselMohawk** [comment]
Me when I don't send my pending reviews by accident :)

File: Resources/Prototypes/_Harmony/Entities/Clothing/Uniforms/jumpsuits.yml [hunk 1]
```diff
@@ -19,3 +19,25 @@
     sprite: _Harmony/Clothing/Uniforms/Jumpsuit/hop_turtle.rsi
   - type: Clothing
     sprite: _Harmony/Clothing/Uniforms/Jumpsuit/hop_turtle.rsi
+
+- type: entity
+  parent: [ClothingUniformBase, BaseRestrictedContraband]
+  id: ClothingUniformJumpsuitWardenBlueHarmony
+  name: Warden's Blue Suit
```
Comment (FluffMe) [inline Resources/Prototypes/_Harmony/Entities/Clothing/Uniforms/jumpsuits.yml:26]: Please make the name all lowercase to match the names of other items in the game

**Review by FluffMe — CHANGES_REQUESTED** [review]
Only two minor changes, and it's good to go!
Thank you very much for working on this.

**Comment by BoskiYourk** [comment]
Capitalization is fixed, so this will hopefully be the last commit needed.

**Review by DieselMohawk — APPROVED** [review]

**Review by FluffMe — APPROVED** [review]

**Pull request merged**